app/routes/titiler/algorithms/dist_alerts.py

```diff
@@ -0,0 +1,27 @@
+from collections import OrderedDict, namedtuple
+from .alerts import Alerts
+
+from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
+
+
+class DISTAlerts(Alerts):
+    title: str = "Land Disturbunce (DIST) Alerts"
+    description: str = "Decode and visualize DIST alerts"
+
+    Colors: namedtuple = namedtuple("Colors", ["red", "green", "blue"])
```

Comment:
Any need to include this if it is the same as the Base Class?

Reply:
Yes, they are not accessible to the subclasses. I ended up taking them out of the class and reusing them both in dist and integrated alerts modules now.